packages/client/components/NewGitLabIssueMenu.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import {MenuProps} from '~/hooks/useMenu'
+import RepoIntegrationGitLabMenuItem from './RepoIntegrationGitLabMenuItem'
+import useSearchFilter from '~/hooks/useSearchFilter'
+import {EmptyDropdownMenuItemLabel} from './EmptyDropdownMenuItemLabel'
+import Menu from './Menu'
+import {SearchMenuItem} from './SearchMenuItem'
+
+interface Props {
+  handleSelectFullPath: (key: string) => void
+  menuProps: MenuProps
+  gitlabProjects: {id: string; fullPath: string}[]
+}
+
+const getValue = (item: {fullPath?: string}) => {
+  return item.fullPath || 'Unknown Project'
+}
+
+const NewGitLabIssueMenu = (props: Props) => {
+  const {handleSelectFullPath, menuProps, gitlabProjects} = props
+
+  const {query, filteredItems: filteredProjects, onQueryChange} = useSearchFilter(
+    gitlabProjects,
+    getValue
+  )
+
+  const onClick = (fullPath: string) => {
```

Comment:
-1 this function doesn't add any value. consider using the pattern in NewGitHubIssueMenu
+1 anything named `onClick` should probably accept a mouse event (or nothing, if the event isn't used)